packages/web/src/components/tipping/tip-audio/SendTip.tsx

```diff
@@ -226,8 +215,11 @@ export const SendTip = () => {
 
   return (
     <div className={styles.container}>
-      {renderProfilePicture()}
-      {renderBecomeTopSupporter()}
+      <TipProfilePicture user={profile} />
+      {!hasError && isFirstSupporter && renderBecomeFirstSupporter()}
```

Comment:
Prefer ternary here and below: https://www.notion.so/audiusproject/Client-Javascript-Styleguide-4f34640139fc4a809c3765131bc597ff#f1c89a4e0843430bbf98f0ce53dbcf14